charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml

```diff
@@ -141,21 +141,84 @@ data:
                 - name: OT_ENABLED
                   value: {{ .Values.application.ray.openTelemetryCollector.enabled | quote }}
 {{- end }}
-            - image: fluent/fluent-bit:1.9.10
-              name: ray-head-logs
+            - image: proxy:local
```

Comment:
a few weeks back we had discussed possibly running this as a separate service rather than a sidecar on each ray cluster, but I don't remember where we landed on that

Reply:
We talked about possibilities.  I think that it may be better as a sidecar because the communication between the ray worker container and the proxy may not be safe so it is better to be an internal communication in the pod instead of between pods. WDYT?

Reply:
sorry, missed this one with all the other threads...

assuming we use Istio to encrypt intracluster traffic, would that be sufficient to mitigate the risk? 

The other thought I had was if some of the complexity (dipping into iptables for example) could be avoided by running as a separate service, something like [this](https://github.com/Qiskit-Extensions/quantum-serverless/pull/1232#discussion_r1544833488) maybe... also leveraging Istio to handle the redirect... just thinking out loud, don't have a strong opinion here (other than the simpler the better :) )